bin/elasticache.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# due to having two routemasters in one hopper app we need a way to
+# set the redis url differently for some of them. this is how we do it.
+set -e
+
+export ROUTEMASTER_REDIS_URL=$ELASTICACHE_ROUTEMASTER_REDIS_URL
+$@
```

Comment:
I think the name of that binary should be changed to something more explicit, like `run_with_elasticache_redis.sh` or something, it's quite hard to know what it does from the name